outboxafrica/edu_lookbook
# Bg remove redundant files

#### What does this PR do?
Removes redundant files from the code base.
#### Description of Task to be completed?
Merge the profile and User crud processes into one Branch.
#### How should this be manually tested?
Fetch remote branches, then checkout to this PR's branch. You then can check for removed files.
#### Any background context you want to provide?
On the first PR I raised about restructuring and fixing broken endpoints in the codebase, I made a wrong squash hence when I reset the squash to point back at the right HEAD, these files were recovered. Pushing the recovered commit ended up pushing the files as well.
